modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -1730,127 +1730,132 @@ public void testActiveTransactions() throws Exception {
         CommandHandler h = new CommandHandler();
 
         final VisorTxInfo[] toKill = {null};
+        IgniteInternalFuture<?> startFut = null;
```

Comment:
Why you have changed the test ? It looks correct to me and doesn't need changing.

Reply:
Unfortunately, the failure of this test can lead to the fact that the whole test suite will hang.

Let's consider the following scenario:
The test starts new transactions and one of them is "blocked" by the `unlockLatch`
     `IgniteInternalFuture<?> fut = startTransactions("testActiveTransactions", lockLatch, unlockLatch, true);`
This method uses the wrong method in order to wait for that latch `U.awaitQuiet(unlockLatch);`
So, if the test fails before this latch is released, then a thread from the pool `GridTestSafeThreadFactory#startedThreads` (which is used by `runMultiThreadedAsync`) has no chance to be completed and this is the cause of hanging the whole test suite.

 It seems to me, that test is absolutely correct and I should only change `U.awaitQuiet(unlockLatch);` to `unlockLatch.await()`.